examples/basic/plot_example_regression.py

```diff
@@ -66,7 +66,7 @@
 scores, model = run_cross_validation(
     X=X, y=y, data=train_diabetes, preprocess_X='zscore',
     problem_type='regression', model='ridge', return_estimator='final',
-    scoring='neg_mean_absolute_error')
+    scoring=['neg_mean_absolute_error'])
```

Comment:
Does it really have to be a list with one element?

Reply:
No, but if its not than the name of the resulting column will be `test_score` instead of `test_neg_mean_absolute_error`.
Se, note above: I made it behave like scikit-learn, but can also make it always use the name of the scorer if one is provided. 
For the highlighted example I could also later change the column  name to `test_score`

Reply:
I find it counter intuitive to give that as an example. Let's keep examples simpler.

Reply:
Now, we do not use the lists, but get them by `['test_scoerer']`

Reply:
Registering has to be adjusted similar to #116